drivers/bridge/bridge.go

```diff
@@ -98,6 +98,7 @@ type bridgeNetwork struct {
 
 type driver struct {
 	config      *configuration
+	configured  bool
```

Comment:
sorry, I was not clear. I was just pointing to the restriction in bridge driver where the `driver.Config()` can be called just once.
Yes. This PR is not introducing that limitation. But am trying to understand if you think this is a good restriction to have ? 
If we agree that this restriction is not valid, then we can approach this PR differently.